README.md

```diff
@@ -395,5 +395,5 @@ This project is in continous development and is striving for feature parity with
 
 ### Contributing.
 
-Your pull request are definitely welcome :heart:! We alway looking to add more test and acheive feature parity with the Buycoins API. Take a look at [Contributing.md](Contributing.md)
+Your pull request are definitely welcome :heart:! We're always looking forward to add more test and acheive feature parity with the Buycoins API. Take a look at [Contributing.md](Contributing.md)
```

Comment:
```suggestion
Your pull requests are definitely welcome :heart:! We're always looking forward to adding more tests and achieving feature parity with the Buycoins API. Take a look at [Contributing.md](Contributing.md)
```

Reply:
Thank you for replying @SomtochiAma i will do exactly that. Thank you so much.